src/execution_plan/ops/op_node_by_id_seek.c

```diff
@@ -90,7 +90,7 @@ static Record NodeByIdSeekConsumeFromChild(OpBase *opBase) {
 
 	Node n = _SeekNextNode(op);
 
-	if(n.attributes == NULL) { // Failed to retrieve a node.
+	if(n.attributes == NULL || n.attributes == &NULL_ATTRIBUTE_SET) { // Failed to retrieve a node.
```

Comment:
I don't believe we need to modify this file, please revert back to original version

Reply:
I removed the comparisons ``n.attributes == &NULL_ATTRIBUTE_SET``, but I think that we need to change ``_SeekNextNode()``.
The function ``_SeekNextNode()`` calls to ```Node n = GE_NEW_NODE();``` and if the program does not enter in the while loop, then the returned value will have ``n.attributes == &NULL_ATTRIBUTE_SET``, which is an unexpected value that is not recognized as ``NULL``.

The function ``_SeekNextNode()``  was modified, please check if it is correct.